applications/teletype/priv/templates/service_added.html

```diff
@@ -64,7 +64,7 @@ <h2 style="margin:0;padding:0;text-align:center;font-family:'Open Sans',sans-ser
                                         <tr>
                                             <td valign="middle" colspan="2" style="padding:10px 0;font-family:'Open Sans',sans-serif;color:#555555;">
                                                 <b>{{srv_item}}</b><br><span style="color:#999999;font-size:13px"><small>Quantity: {{item_props.quantity}}<br>Rate: {{item_props.rate}}{% if item_props.discounts != "0.00" %}<br>Discounts: {{item_props.discounts}}{% endif %}{% if item_props.changes
-                                                %}<br>{% with type=item_props.changes.type %}{% if type == "plan_new" %}<span style="color:#07861d;">Newly started billed by your service plan</span>{% elif type == "plan_removed" %}<span style="color:#de0a0a">{{item_props.changes.quantity}} item(s) stopped billed by your service plan</span>{% elif type == "added" %}<span style="color:#07861d">{{item_props.changes.quantity}} item(s) added to your service</span>{% elif type == "removed" %}<span style="color:#de0a0a">{{item_props.changes.quantity}} item(s) removed from your service</span></small></span>{% endif %}{% endwith %}{% endif %}
+                                                %}<br>{% with type=item_props.changes.type %}{% if type == "plan_new" %}<span style="color:#07861d;">New service added to your plan</span>{% elif type == "plan_removed" %}<span style="color:#de0a0a">{{item_props.changes.quantity}} item(s) stopped billed by your service plan</span>{% elif type == "added" %}<span style="color:#07861d">{{item_props.changes.quantity}} item(s) added to your service</span>{% elif type == "removed" %}<span style="color:#de0a0a">{{item_props.changes.quantity}} item(s) removed from your service</span></small></span>{% endif %}{% endwith %}{% endif %}
```

Comment:
@icehess 
Should update the verbiage for removal of a service plan item as well.

Currently:
"{{item_props.changes.quantity}} item(s) stopped billed by your service plan"
should make it match the added, perhaps:
"Service removed from your plan"